packages/components/app/styles/components/table.scss

```diff
@@ -16,32 +17,53 @@
 
 .hds-table--striped {
   tbody {
-    tr:nth-child(odd) {
+    tr:nth-child(even) {
       background-color: var(--token-color-surface-faint);
     }
   }
 }
 
+.hds-table--density-short td {
```

Comment:
since in the other PR we're using `.hds-table__tbody td` as selector, to avoid risks of collision in specificity I would suggest to use `.hds-table--density-short .hds-table__tbody td` here (it works also without, because we're changing different props, but is safer)

Reply:
is it any riskier? we don't have td elements in the thead element.

Reply:
`.hds-table--density-short td` has the same specificity as `.hds-table__tbody td`, so there are risks of conflict. if the "baseline" of the `TD` is declared as `.hds-table__tbody td` then `.hds-table--density-short` is a specialized version of the `TD` which means its selector should be `.hds-table--density-short hds-table__tbody td`

Reply:
this has been changed in PR #658

Reply:
OK, I'll review/check it there.Thanks